c7n/tags.py

```diff
@@ -336,6 +336,24 @@ class Tag(Action):
         key={'type': 'string'},
         value={'type': 'string'},
         tag={'type': 'string'},
+        Tags={
```

Comment:
i'm still noodling on the user exposed schema syntax here, ie lower case tags, object instead of array, but that doesn't quite cover off on type expression. wondering if we could do something interesting with format strings, which would also cover on other extensions and would allow just a k/v object instead of array perhaps. ie. see https://pyformat.info/

Reply:
we already have a 'tags' parameter, adding a separate 'Tags' is more than confusing.